SamplesDashboard/SamplesDashboard/Services/MicrosoftOpenSourceService.cs

```diff
@@ -0,0 +1,86 @@
+// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Identity.Client;
+using Newtonsoft.Json;
+using SamplesDashboard.Models;
+
+namespace SamplesDashboard.Services
+{
+    /// <summary>
+    /// This class contains query services for the Microsoft Open Source GitHub portal
+    /// </summary>
+    public class MicrosoftOpenSourceService
+    {
+        private readonly IHttpClientFactory _clientFactory;
+        private readonly IConfiguration _config;
+        private readonly ILogger<MicrosoftOpenSourceService> _logger;
+        private readonly IConfidentialClientApplication _cca;
+        private static readonly string[] _scopes =
+            { "api://5bc5e692-fe67-4053-8d49-9e2863718bfb/.default" };
```

Comment:
anything special to know about this guid?  what does it represent.

Reply:
It's the app ID of the internal API. I'll add it as a constant and build the string from it.

Reply:
Or no, I'll just comment the `_scopes` property.

Reply:
Fixed in latest push, I'll leave it to you to resolve if you agree with my fix :D.